ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -103,14 +104,39 @@ check_mechanism_against_processing(struct pkcs11_session *session,
 		    !session->processing->relogged)
 			return PKCS11_CKR_USER_NOT_LOGGED_IN;
 
+		if (session->processing->oneshotted) {
+			EMSG("Cannot perform update on one-shot processing");
```

Comment:
`update step` instead of plain `update`

Reply:
OK. I suppose `one-shot` should also be `oneshot`.